src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -2441,13 +2443,13 @@ void KaleidoScope_LoadDungeonMap(PlayState* play) {
     InterfaceContext* interfaceCtx = &play->interfaceCtx;
     s32 pad;
 
-    DmaMgr_SendRequest1(interfaceCtx->mapSegment,
-                        (uintptr_t)_map_48x85_staticSegmentRomStart + (R_MAP_TEX_INDEX * 2040), 2040,
-                        "../z_kaleido_scope_PAL.c", 3467);
+    DmaMgr_SyncDmaRequestDebug(interfaceCtx->mapSegment,
+                               (uintptr_t)_map_48x85_staticSegmentRomStart + (R_MAP_TEX_INDEX * 2040), 2040,
+                               "../z_kaleido_scope_PAL.c", 3467);
 
-    DmaMgr_SendRequest1(interfaceCtx->mapSegment + 0x800,
-                        (uintptr_t)_map_48x85_staticSegmentRomStart + ((R_MAP_TEX_INDEX + 1) * 2040), 2040,
-                        "../z_kaleido_scope_PAL.c", 3471);
+    DmaMgr_SyncDmaRequestDebug(interfaceCtx->mapSegment + 0x800,
+                               (uintptr_t)_map_48x85_staticSegmentRomStart + ((R_MAP_TEX_INDEX + 1) * 2040), 2040,
```

Comment:
0x7F8's a pretty weird number but should probably still be in hex given its closeness to the 0x800.